decidim-initiatives/app/views/decidim/initiatives/initiative_signatures/finish.html.erb

```diff
@@ -1,6 +1,23 @@
-<%= cell("decidim/announcement", t("decidim.initiatives.initiative_votes.create.success_html", title: translated_attribute(current_initiative.title)), callout_class: "secondary") %>
+<div class="flex flex-col gap-4 items-center text-center my-12">
+  <div class="success-image">
+    <svg class="animation-success__checkmark" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 52 52">
+      <circle class="animation-success__checkmark__circle" cx="26" cy="26" r="25" fill="none" />
+      <path class="animation-success__checkmark__check" fill="none" d="M14.1 27.2l7.1 7.2 16.7-16.8" />
+    </svg>
```

Comment:
wondering if we could extract the icon so it can be reused in the future.

Reply:
You are right. Done!